port/ztpf/omrsignal_context.c

```diff
@@ -343,13 +343,13 @@ ztpfDeriveUcontext( ucontext_t *uscPtr ) {
    DIB *pDib = ecbp2()->ce2dib;	 
    DBFHDR *pJdb = (DBFHDR *)(pDib->dibjdb);
    DBFITEM *pDbi = NULL;
-   uint8_t*	lowcore = NULL;
+   void*	lowcore = NULL;

```

Comment:
Another minor nitpick.  I know you're just changing the type here but can you also fix the indenting so that the declaration is:
``` 
void * lowcore = NULL;
```
No tab between the * and variable?  Thanks.